solr/solrj/src/resources/java-template/api.mustache

```diff
@@ -133,7 +133,6 @@ public class {{classname}} {
                 {{#isBodyParam}}
                 {{#vendorExtensions.x-genericEntity}}
                     this.requestBody = requestBody;
-                    addHeader("Content-type", "application/json");
```

Comment:
This doesn't belong here.  It's the ContentWriter that's responsible for the type.

Reply:
I do like that better from an abstraction perspective, but I remember putting this line here only after noticing that relying on the ContentWriter was leaving the header unset in some cases.  I wish I remembered the context there better.

But let's not worry about that now - if it crops up again I'll reinvestigate at that point...